code/modules/holiday/holidays.dm

```diff
@@ -398,75 +315,32 @@
 // JULY
 
 /datum/holiday/doctor
-	name = "Doctor's Day"
+	name = "Doktorlar Günü"
 	begin_day = 1
 	begin_month = JULY
 	holiday_hat = /obj/item/clothing/head/costume/nursehat
 
 /datum/holiday/ufo
-	name = "UFO Day"
+	name = "UFO Günü"
 	begin_day = 2
 	begin_month = JULY
 	holiday_hat = /obj/item/clothing/head/collectable/xenom
 
 /datum/holiday/ufo/getStationPrefix() //Is such a thing even possible?
 	return pick("Ayy","Truth","Tsoukalos","Mulder","Scully") //Yes it is!
 
-/datum/holiday/usa
-	name = "US Independence Day"
-	timezones = list(TIMEZONE_EDT, TIMEZONE_CDT, TIMEZONE_MDT, TIMEZONE_MST, TIMEZONE_PDT, TIMEZONE_AKDT, TIMEZONE_HDT, TIMEZONE_HST)
-	begin_day = 4
-	begin_month = JULY
-	mail_holiday = TRUE
-	holiday_hat = /obj/item/clothing/head/cowboy/brown
-	holiday_colors = list(
-		COLOR_OLD_GLORY_BLUE,
-		COLOR_OLD_GLORY_RED,
-		COLOR_WHITE,
-		COLOR_OLD_GLORY_RED,
-		COLOR_WHITE,
-	)
-
-
-/datum/holiday/usa/getStationPrefix()
-	return pick("Independent","American","Burger","Bald Eagle","Star-Spangled", "Fireworks")
-
-/datum/holiday/writer
-	name = "Writer's Day"
-	begin_day = 8
-	begin_month = JULY
-
-/datum/holiday/france
-	name = "Bastille Day"
-	timezones = list(TIMEZONE_CEST)
-	begin_day = 14
+/datum/holiday/demokrasi_bayrami
+	name = "Milli Birlik ve Demokrasi Günü"
+	begin_day = 15
 	begin_month = JULY
-	holiday_hat = /obj/item/clothing/head/beret
-	mail_holiday = TRUE
 	holiday_colors = list(
-		COLOR_FRENCH_BLUE,
+		COLOR_TURKISH_RED,
 		COLOR_WHITE,
-		COLOR_FRENCH_RED
 	)
-	holiday_pattern = PATTERN_VERTICAL_STRIPE
-
-/datum/holiday/france/getStationPrefix()
-	return pick("Francais", "Fromage", "Zut", "Merde", "Sacrebleu")
-
-/datum/holiday/france/greet()
-	return "Do you hear the people sing?"
-
-/datum/holiday/hotdogday
-	name = HOTDOG_DAY
-	begin_day = 17
-	begin_month = JULY
-
-/datum/holiday/hotdogday/greet()
-	return "Happy National Hot Dog Day!"
 
 //Gary Gygax's birthday, a fitting day for Wizard's Day
 /datum/holiday/wizards_day
-	name = "Wizard's Day"
+	name = "Wizard'ın Günü"  // ozel isim gibi
```

Comment:
```suggestion
	name = "Büyücüler Günü"
```
Doctor's Day nasıl Doktorlar Günü ise bu da Büyücüler Günü olmaz mı?

Reply:
Gary Gygax'in takma ismi wizard